CHANGELOG.rst

```diff
@@ -2428,6 +2427,7 @@ Deprecated
 Fixed
 ~~~~~
 
+* Fix ssh zombies when using ProxyCommand from ssh config #4881 [Eric Edgar]
```

Comment:
Please check the `CHANGELOG.rst` file.

You're including the Changelog record for the st2 `v1.4.0` version. It should go into _in development_ instead.

Reply:
This should be resolved now.